components/layout/Footer.tsx

```diff
@@ -7,10 +7,20 @@ const Footer = () => {
   return (
     <FooterWrapper>
       <ContentWrapper>
-        Powered By
-        <HighlightLink>
-          <Link href="https://chingu.io/">Chingu</Link>
-        </HighlightLink>
+        <div>
+          Powered By
+          <HighlightLink>
+            <Link href="https://chingu.io/">Chingu</Link>
```

Comment:
Should we add "target="_blank" here? .. It's a good practice to add that whenever you link out of the current site